utils.js

```diff
@@ -18,6 +18,14 @@ export const variantValidator = (variant) => {
   return COLOR_VARIANTS.includes(variant) !== -1;
 };
 
+export function humanize(str) {
+  if (typeof str === 'string') {
```

Comment:
so it's already here! just gotta import it and pass it to the methods section `methods: { humanize, ...}` then can delete all those inline versions